app/src/main/java/com/github/gotify/login/LoginActivity.java

```diff
@@ -42,6 +51,24 @@
     @BindView(R.id.password)
     EditText passwordField;
 
+    @BindView(R.id.sslGroup)
+    LinearLayout sslGroup;
```

Comment:
Sorry for not realizing this in the first round of the review. 
I think it would be better, when the advanced settings would be in a dialog. Like this, they are not bloating the login activity, I got a working example here, https://github.com/gotify/android/commit/84861df2a648c36100e40d7b73b6b9f34d3028bf 

Stuff that additionally needs to be done: 
1. Remove unused buttons/views in LoginActivity
2. Use R.string in AdvancedDialog

You can either copy the stuff from the commit or cherry pick it.

Reply:
Cherry-picked this (nice code!) and cleaned up unused views / switched to string resources